ewatercycle/models/lisflood.py

```diff
@@ -15,165 +12,183 @@
 from grpc4bmi.bmi_client_singularity import BmiClientSingularity
 
 from ewatercycle import CFG
-from ewatercycle.forcing.forcing_data import ForcingData
+from ewatercycle.forcing.lisflood import LisfloodForcing
 from ewatercycle.models.abstract import AbstractModel
 from ewatercycle.parametersetdb.config import AbstractConfig
+from ewatercycle.util import get_time
 
 
 @dataclass
 class LisfloodParameterSet:
-    """Input files specific for parameterset, model boundaries, and configuration template files
+    """Input files specific for parameter_set, model boundaries, and configuration template files
 
     Example:
 
     .. code-block::
 
-        parameterset = LisfloodParameterSet(
-            root=Path('/projects/0/wtrcycle/comparison/lisflood_input/Lisflood01degree_masked'),
-            mask=Path('/projects/0/wtrcycle/comparison/recipes_auxiliary_datasets/LISFLOOD/model_mask.nc'),
-            config_template=Path('/projects/0/wtrcycle/comparison/lisflood_input/settings_templates/settings_lisflood.xml'),
-            lisvap_config_template=Path('/projects/0/wtrcycle/comparison/lisflood_input/settings_templates/settings_lisvap.xml'),
+        parameter_set = LisfloodParameterSet(
+            PathRoot='/projects/0/wtrcycle/comparison/lisflood_input/Lisflood01degree_masked',
+            MaskMap='/projects/0/wtrcycle/comparison/recipes_auxiliary_datasets/LISFLOOD/model_mask.nc',
+            config_template='/projects/0/wtrcycle/comparison/lisflood_input/settings_templates/settings_lisflood.xml',
         )
     """
-    root: PathLike
+    PathRoot: Path
     """Directory with input files"""
-    mask: Path
+    MaskMap: Path
     """A NetCDF file with model boundaries"""
-    config_template: PathLike
+    config_template: Path
     """Config file used as template for a lisflood run"""
-    lisvap_config_template: Optional[PathLike] = None
-    """Config file used as template for a lisvap run"""
-
 
-# Mapping from lisvap output to lisflood input files. MAPS_PREFIXES dictionary
-# contains lisvap filenames in lisvap and lisflood config files and their
-# equivalent cmor names
-MAPS_PREFIXES = {
-    'E0Maps': {'name': 'PrefixE0', 'value': 'e0'},
-    'ES0Maps': {'name': 'PrefixES0', 'value': 'es0'},
-    'ET0Maps': {'name': 'PrefixET0', 'value': 'et0'},
-}
+    def __setattr__(self, name: str, value: Union[str, Path]):
+        self.__dict__[name] = Path(value).expanduser().resolve()
 
 
 class Lisflood(AbstractModel):
     """eWaterCycle implementation of Lisflood hydrological model.
 
+    Args:
+    version: pick a version for which an ewatercycle grpc4bmi docker image is available.
+    parameter_set: LISFLOOD input files. Any included forcing data will be ignored.
+    forcing: a LisfloodForcing object.
```

Comment:
Args not rendered correctly in Sphinx docs
```suggestion
      version: pick a version for which an ewatercycle grpc4bmi docker image is available.
      parameter_set: LISFLOOD input files. Any included forcing data will be ignored.
      forcing: a LisfloodForcing object.
```